pkg/apis/core/types.go

```diff
@@ -635,6 +696,16 @@ type PersistentVolumeClaimStatus struct {
 	// +mapType=granular
 	// +optional
 	AllocatedResourceStatuses map[ResourceName]ClaimResourceStatus
+	// currentVolumeAttributesClassName is the current name of the VolumeAttributesClass the PVC is using.
```

Comment:
Add a note "When unset, there is no VolumeAttributeClass applied to this PersistentVolumeClaim"